sources/platform/monitoring/index.md

```diff
@@ -1,94 +1,108 @@
+2. **Number of fields** returned. This is something that indicates a change of the website. For example the manufacturer name moved to other place.
 
-## Notifications {#notifications}
+**Performance**:
 
-The suite will send notifications to the [email address associated with your account](https://console.apify.com/account) by default.
+1. **Duration** of the run. If your solution is taking longer than usual to finish, you can set up an alert to notify you, this will help you prevent your solution from being stuck and from wasting resources.
 
-You can disable email notifications or change the email address in a task **Input**'s **Email notifications** section.
+**Usage and cost**:
 
-![Customize notifications](./images/customize-notifications.png)
+1. **Usage cost** may change when the robot blocking solution gets implemented. An increase of the cost may indicate that many URLs are being retried.
+2. **Proxy usage.** Seeing how your solution is using proxy and if there are any changes can help you optimize your usage and prevent increased costs if your solution starts behaving differently than you expect.
 
-If you use [Slack](https://slack.com/), we suggest using Slack notifications instead of email. More details and setup instructions [here](https://apify.com/apify/monitoring#notifications).
+These are just a few examples of what you can monitor. It's always recommended to start small, iterate and become more complex over time based on the experience.
 
-### Example of a failed run {#example-of-a-failed-run}
+### Alternative solutions
 
-Below is an example of an email notification for a task run that did not return enough results.
+For more complex monitoring you can use the [monitoring suite](https://apify.com/apify/monitoring) which is a collection of [Apify actors](../actors/index.md) that allows you to automate the monitoring of jobs you have running on the [Apify platform](https://apify.com). Compared to the monitoring in the Apify Console, the monitoring suite offers some features that are not **currently** available in the Apify Console, such as:
 
-![Failed run example](./images/puppies-failed-run.png)
+1. Schema validation of the output
+2. Duplicates check in the output
+3. Dashboards with data grouping
+4. Running the monitoring daily/weekly/monthly instead of after every run
 
-To see what went wrong, click on the **time** next to **Actor run** to open your task's **Runs** dashboard. When you click on **Details**, you'll find a detailed monitoring report in JSON format.
+> But be warned that this solution is more complex and requires more time to set up. Also, it uses schedules, actors, tasks, and webhooks, so using it will increase your overall usage on the Apify platform.
```

Comment:
```suggestion
> Please note that this solution is more complex and requires more time to set up. Also, it uses schedules, Actors, tasks, and webhooks, so using it will increase your overall usage on the Apify platform.
```